server/graphql/v2/input/PaymentMethodInput.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLBoolean, GraphQLInputFieldConfigMap, GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import { PaymentMethodType } from '../enum';
+
+import { CreditCardCreateInput } from './CreditCardCreateInput';
+import { PaypalPaymentDataInput } from './PaypalPaymentDataInput';
+
+export const PaymentMethodInput = new GraphQLInputObjectType({
+  name: 'PaymentMethodInput',
+  description: 'An input to use for creating or retrieving payment methods',
+  fields: (): GraphQLInputFieldConfigMap => ({
+    id: {
+      type: GraphQLString,
+      description: 'The id assigned to the payment method',
+    },
+    type: {
+      type: PaymentMethodType,
+      description: 'Type of this payment method',
+    },
+    name: {
+      type: GraphQLString,
+      description: 'Name of this payment method',
+    },
+    save: {
```

Comment:
Maybe we could take the opportunity to make this one more explicit, ie: `saveForLater`.

Reply:
The field is not yet defined on `PaymentMethod`, so now is a good time to rename it if we want to. I personally like to prefix booleans with `is`/`has`, so `isSavedForLater`.